mdbook/src/tutorials/policy-management.md

```diff
@@ -96,3 +96,33 @@ After the following command is executed, rate limit policy is no longer applied.
 ```
 cargo run  --release --bin addonctl -- --config eval/policy/ratelimit/detach.toml --pid 2012290 --sid 1
 ```
+
+# Semantics
+
+The engines can form a graph, and is connected via tx/rx channels, which are bidirectional. 
```

Comment:
Do you mean tx+rx are bidirectional?

Reply:
Maybe `The engines can form a graph, and are connected via unidirectional tx/rx channels.`?

Reply:
Sure, I will fix that.